measures/measures-schema.yaml

```diff
@@ -197,3 +205,55 @@ definitions:
       - thoracicSurgery
       - urology
       - vascularSurgery
+
+  eligibilityOption:
+    type: object
+    properties:
+      submissionMethods: { $ref: #/definitions/qualityCodesSubmissionMethods }
+      sexCode:
+        enum: [M, F]
+      minAge:
+        type: integer
+      maxAge:
+        type: integer
+      diagnosisCodes: { $ref: #/definitions/arrayOfStringIdentifiers }
+      additionalDiagnosisCodes: { $ref: #/definitions/arrayOfStringIdentifiers }
+      procedureCodes:
+        type: array
+        items: { $ref: #/definitions/codeObject }
+    required: [submissionMethods]
+
+  performanceOption:
+    type: object
+    properties:
+      submissionMethods: { $ref: #/definitions/qualityCodesSubmissionMethods }
+      optionType:
+        enum: [performanceMet, performanceNotMet, performanceExclusion]
+      qualityCodes:
+        type: array
+        items: { $ref: #/definitions/codeObject }
+    required: [submissionMethods, optionType, qualityCodes]
+
+  codeObject:
+    type: object
+    properties:
+      code:
+        type: string
+      modifiers:  { $ref: #/definitions/arrayOfStringIdentifiers }
```

Comment:
There's a limited set in use at the moment (more than 4) but there is I believe a huge space of possible modifiers. (see [Modifier Reference Guide](https://www.sccma-mcms.org/portals/19/assets/docs/modifier-reference-guide.pdf)) but perhaps @akegan can weigh in on if it makes sense to limit the possible set.